src/main/io/adsb.c

```diff
@@ -131,6 +131,11 @@ void gpsDistanceCmBearing(int32_t currentLat1, int32_t currentLon1, int32_t dest
     *bearing = wrap_36000(*bearing);
 };
 
+bool adsbHeartbeat(void){
+    adsbVehiclesStatus.heartbeatMessagesTotal++;
```

Comment:
do we need to count the heartbeats?
If we do,  I don't think adsbVehicleStatus is the right place for it, as it is not an adsb message.